### PR TITLE
GitHub Releases List Command

### PR DESCRIPTION
**Pull request by tomneeman151293**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/40601

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/GitHub/Integrations/GitHub/GitHub.py [hunk 1]
```diff
@@ -1885,6 +1885,41 @@ def get_path_data():
...
+    """
+    Gets releases data of given repository in given organization.
+    Returns:
+        CommandResults data.
+    """
+    args: Dict[str, Any] = demisto.args()
+
+    repo: str = args.get('repository') or REPOSITORY
+    organization: str = args.get('organization') or USER
+    limit = int(args.get('limit', 30))
```
Comment (dantavori) [inline Packs/GitHub/Integrations/GitHub/GitHub.py:1898]: please use our standard for commands with pagination:
https://xsoar.pan.dev/docs/integrations/code-conventions#pagination-in-integration-commands

File: Packs/GitHub/Integrations/GitHub/GitHub.py [hunk 2]
```diff
@@ -1940,7 +1975,8 @@ def fetch_incidents_command():
     'GitHub-create-release': create_release_command,
     'Github-list-issue-events': get_issue_events_command,
     'GitHub-add-issue-to-project-board': add_issue_to_project_board_command,
-    'GitHub-get-path-data': get_path_data
+    'GitHub-get-path-data': get_path_data,
+    'GitHub-releases-list': github_releases_list_command
```
Comment (dantavori) [inline Packs/GitHub/Integrations/GitHub/GitHub.py:1979]:
```suggestion
    'GitHub-releases-list': github_releases_list_command,
```

File: Packs/GitHub/Integrations/GitHub/GitHub.yml [hunk 3]
```diff
@@ -3920,6 +3920,133 @@ script:
...
+      required: false
+      secret: false
+    - default: false
+      description: The repository containing the releases. Defaults to repository instance parameter if not given.
+      isArray: false
+      name: repository
+      required: false
+      secret: false
+    deprecated: false
+    description: Get releases data from given repository and organization.
```
Comment (dantavori) [inline Packs/GitHub/Integrations/GitHub/GitHub.yml:3944]:
```suggestion
    description: Gets releases data from a given repository and organization.
```

**Comment by tomneeman151293** [comment]
@dantavori Fixed notes

**Review by dantavori — APPROVED** [review]

**Pull request merged**